spark-2.4/src/test/scala/org/neo4j/spark/SparkConnectorAuraTest.scala

```diff
@@ -0,0 +1,87 @@
+package org.neo4j.spark
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.SparkSession
+import org.junit.Assert._
+import org.junit.Assume.assumeTrue
+import org.junit.{AfterClass, Before, BeforeClass, Test}
+import org.neo4j.driver._
+import org.neo4j.spark.SparkConnectorAuraTest._
+
+object SparkConnectorAuraTest {
+  private var neo4j: Driver = _
+  private val username: Option[String] = Option[String](System.getenv("AURA_USER"))
+  private val password: Option[String] = Option[String](System.getenv("AURA_PASSWORD"))
+  private val url: Option[String] = Option[String](System.getenv("AURA_URI"))
+
+  val sparkSession = SparkSession.builder()
```

Comment:
please create the `sparkSession` after the assume checks in the `BeforeClass` and tear it down in the `AfterClass`

Reply:
also in spark 3.0